packages/app/src/modules/options.js

```diff
@@ -119,11 +119,13 @@ export const getOptionsFromVisualization = visualization => {
     }
 
     // cast option values from Number for some options
-    ;['sortOrder', 'topLimit'].forEach(option => {
-        if (Object.prototype.hasOwnProperty.call(visualization, option)) {
-            optionsFromVisualization[option] = String(visualization[option])
+    ;['baseLineValue', 'targetLineValue', 'sortOrder', 'topLimit'].forEach(
+        option => {
+            if (Object.prototype.hasOwnProperty.call(visualization, option)) {
```

Comment:
Why is this not just `if (visualization.hasOwnProperty(option))` ?

Reply:
The linter complains about that form.

Reply:
Do we get e.g. https://eslint.org/docs/rules/no-prototype-builtins from cli-style?